src/replication.c

```diff
@@ -2968,7 +3774,10 @@ void replicationSetPrimary(char *ip, int port) {
         moduleFireServerEvent(VALKEYMODULE_EVENT_PRIMARY_LINK_CHANGE, VALKEYMODULE_SUBEVENT_PRIMARY_LINK_DOWN, NULL);
 
     server.repl_state = REPL_STATE_CONNECT;
-    serverLog(LL_NOTICE, "Connecting to PRIMARY %s:%d", server.primary_host, server.primary_port);
+    /* Allow trying rdb-connection sync with the new primary. If newprimaryr doesn't
+     * support rdb-connection sync, we will set to 0 afterwards. */
+    server.primary_supports_dual_conn_sync = -1;
```

Comment:
I actually didn't find where `primary_supports_dual_conn_sync` is referenced? is this still needed?